pkg/splunk/enterprise/util.go

```diff
@@ -1044,11 +1245,20 @@ func initAndCheckAppInfoStatus(client splcommon.ControllerClient, cr splcommon.M
 				return err
 			}
 
+			// TODO: gaurav, this needs to be removed eventually in favor of downloading apps locally
 			_, _, err = ApplyAppListingConfigMap(client, cr, appFrameworkConf, appStatusContext.AppsSrcDeployStatus)
```

Comment:
Why are we not already removing/commenting this now? Wouldn't he configMap update interfere with Phase 3 functioning?

Reply:
We will remove this once we have all the pieces ready to install the app/s via operator pod. We are just in the download stage at the moment.